Homework_1/hw_task_2.py

```diff
@@ -0,0 +1,19 @@
+"""Напишите код, который запрашивает число и сообщает является ли оно простым или составным.
+Используйте правило для проверки: «Число является простым, если делится нацело только на единицу
+и на себя». Сделайте ограничение на ввод отрицательных чисел и чисел больше 100 тысяч."""
+
+num = int(input('Введите число от 1 до 100000: '))
+
+if 1 > num or num > 100000:
```

Comment:
Во-первых, 0 вводиться может, так как он не является отрицательным числом Во-вторых, запись можно сделать проще.
```suggestion
if not 0 <= num <= 100000:
```

Reply:
Но 0 не является  простым числом.